src/core/Radio/Radio.tsx

```diff
@@ -80,7 +80,7 @@ export const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
           style={checkmarkStyle}
         >
           <svg viewBox='0 0 16 16' aria-hidden='true' focusable='false'>
-            <circle cx='8' cy='8' r='6' />
+            <circle cx='8' cy='8' r='4' />
```

Comment:
Could you please change one image so we have checked state tested and avoid such issues in future?

Reply:
same with checkbox, I guess

Reply:
I don't have Docker Desktop installed anymore ☹️ Maybe you or @bentleyvk could do it tomorrow during your work hours. If not, I'll reinstall Docker tomorrow

Reply:
Done